developers/integrations/cloud-hyperscalers/aws/aws.md

```diff
@@ -1,4 +1,19 @@
 ---
-title: AWS
+title: Amazon Web Services
 sidebar_position: 1
----
\ No newline at end of file
+---
+
+AWS (Amazon Web Services) enables you to launch a Weaviate cluster in their marketplace directly, and they support model provider integrations through Sagemaker and Bedrock. 
+
+## AWS and Weaviate
+Weaviate has integrations with [AWS](https://aws.amazon.com/) infrastructure and services like [SageMaker](https://aws.amazon.com/sagemaker/) and [Bedrock](https://aws.amazon.com/bedrock/).
+
+* [Deploying Weaviate on AWS Marketplace](/developers/weaviate/installation/aws-marketplace)
+* [Running embedding and generative models on SageMaker and Bedrock](/developers/weaviate/model-providers/aws)
+
+## Our Resources 
+**Hands on Learning**: Content framed to build your technical understanding with end-to-end tutorials.
+
+| Topic | Details |
+| --- | --- |
+| **RAG with Cohere models on Amazon Bedrock and Weaviate** | The example use case generates targeted advertisements for vacation stay listings based on a target audience. [Notebook](https://github.com/weaviate/recipes/blob/main/integrations/cloud-hyperscalers/aws/RAG_Cohere_Weaviate_v4_client.ipynb).
```

Comment:
It's nice to highlight there is a notebook because sometimes there are two resources (i.e. [here](https://weaviate.io/developers/weaviate/more-resources/dspy#hands-on-learning)). Maybe I can make a third column `Resources` and have the notebook plus another link?